app/models/manageiq/providers/autosde/storage_manager.rb

```diff
@@ -12,6 +12,7 @@ class ManageIQ::Providers::Autosde::StorageManager < ManageIQ::Providers::Storag
   require_nested :StorageService
   require_nested :VolumeMapping
 
+  supports :authentication_status
```

Comment:
I didn't find any. a few calls to the method, but none calling `validate_` or `supports?`

The [`FeatureButton`](https://github.com/kbrock/manageiq-ui-classic/blob/337a8cc0cec51668ffcfee276435c89a7bb3a106/app/helpers/application_helper/button/generic_feature_button.rb) concerns me. I do not understand it but it always makes me wonder if it will sneak a supports into there.

Ideas how to know if this will call that method?
Would we put a button around `authentication_status`?